src/containers/ReviewModal/ReviewActions.jsx

```diff
@@ -36,7 +35,7 @@ export const ReviewActions = ({
         <SubmissionNavigation />
       </div>
     </ActionRow>
-  </div>
+    </div>
```

Comment:
This is separate from the rubric container fix.
Also, why the added indentation?

Reply:
This commit has nothing to do with rubric container at all. It is mostly just updating loose end from the merge.

Reply:
first concern resolved.  Still an extra indentation added :-p